templates/thank-you.html

```diff
@@ -45,7 +45,7 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <p class="ons-u-mb-s">{{ _("Your answers will be processed in the next few weeks.") }}
       {{ _("We may contact you to query your answers via phone or secure message.") }}</p>
     <p class="ons-u-mb-s">{{ _("For more information on how we use this data.") }}<br>
-      <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
+      <a href="https://www.ons.gov.uk/surveys">{{ _("Learn more about how we use this data") }}</a></p>
```

Comment:
Isn't the text above supposed to change to match the text on the ticket?

Reply:
Yeah, description was slightly confusing, added now, thanks.

Reply:
I confirmed with Donna this should be the only text above the link.